eng/pipelines/pipeline-resources.yml

```diff
@@ -74,8 +74,8 @@ extends:
         image: mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.19-WithNode
         options: --cap-add=SYS_PTRACE
 
-      test_debian_11_amd64:
-        image: mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-amd64
+      test_debian_12_amd64:
+        image: mcr.microsoft.com/dotnet-buildtools/prereqs:debian-12-amd64
```

Comment:
This image tag doesn't exist. This is related to the conversation at https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1276. It's not yet known whether we will even create a `debian-12-amd64` image and instead update the existing `debian-11-amd64` to a different image type, like Helix.